streams/jp.m3u

```diff
@@ -41,6 +41,10 @@ https://n24-cdn-live.ntv.co.jp/ch01/index.m3u8
 https://n24-cdn-live.ntv.co.jp/ch02/index.m3u8
 #EXTINF:-1 tvg-id="ShopChannel.jp",Shop Channel (1080p) [Not 24/7]
 https://stream3.shopch.jp/HLS/master.m3u8
+#EXTINF:-1 tvg-id="JORXDTV.jp",TBS (540p)
+https://tbs4.mov3.co/hls/tbs.m3u8
+#EXTINF:-1 tvg-id="",TBS (540p)
+https://tbs5.mov3.co/hls/tbs.m3u8
```

Comment:
![image](https://github.com/user-attachments/assets/79e8fd11-29ac-47bc-aa46-89f57e1e1eca)
not so fast ahaha

Reply:
well lol, that host was known for restreaming many channels... now we know that's pages hahaha

Reply:
It came back it seems.

Reply:
Compared to an original source... that's day and night.
![vlcsnap-2024-11-20-12h17m42s944](https://github.com/user-attachments/assets/80b2cb50-9d55-4488-9dd7-e244d77cc947)